koku/masu/database/presto_sql/reporting_ocpazurecostlineitem_daily_summary.sql

```diff
@@ -51,71 +145,103 @@ WITH cte_ocp_on_azure_resource_id_joined AS (
-        ocp.namespace,
-        ocp.data_source,
-        max(ocp.node) as node,
-        max(json_format(ocp.pod_labels)) as pod_labels,
-        sum(ocp.pod_usage_cpu_core_hours) as pod_usage_cpu_core_hours,
-        sum(ocp.pod_request_cpu_core_hours) as pod_request_cpu_core_hours,
-        sum(ocp.pod_limit_cpu_core_hours) as pod_limit_cpu_core_hours,
-        sum(ocp.pod_usage_memory_gigabyte_hours) as pod_usage_memory_gigabyte_hours,
-        sum(ocp.pod_request_memory_gigabyte_hours) as pod_request_memory_gigabyte_hours,
-        max(ocp.node_capacity_cpu_cores) as node_capacity_cpu_cores,
-        max(ocp.node_capacity_cpu_core_hours) as node_capacity_cpu_core_hours,
-        max(ocp.node_capacity_memory_gigabytes) as node_capacity_memory_gigabytes,
-        max(ocp.node_capacity_memory_gigabyte_hours) as node_capacity_memory_gigabyte_hours,
-        max(ocp.cluster_capacity_cpu_core_hours) as cluster_capacity_cpu_core_hours,
-        max(ocp.cluster_capacity_memory_gigabyte_hours) as cluster_capacity_memory_gigabyte_hours,
-        max(ocp.persistentvolumeclaim) as persistentvolumeclaim,
-        max(ocp.persistentvolume) as persistentvolume,
-        max(ocp.storageclass) as storageclass,
-        max(json_format(volume_labels)) as volume_labels,
-        max(ocp.persistentvolumeclaim_capacity_gigabyte) as persistentvolumeclaim_capacity_gigabyte,
-        max(ocp.persistentvolumeclaim_capacity_gigabyte_months) as persistentvolumeclaim_capacity_gigabyte_months,
-        sum(ocp.volume_request_storage_gigabyte_months) as volume_request_storage_gigabyte_months,
-        sum(ocp.persistentvolumeclaim_usage_gigabyte_months) as persistentvolumeclaim_usage_gigabyte_months
+    max(cast(coalesce(azure.quantity, azure.usagequantity) as decimal(24,9))) as usage_quantity,
+    max(coalesce(billingcurrencycode, currency)) as currency,
+    max(cast(coalesce(azure.costinbillingcurrency, azure.pretaxcost) as decimal(24,9))) as pretax_cost,
+    max(cast(coalesce(azure.costinbillingcurrency, azure.pretaxcost) as decimal(24,9))) * cast({{markup}} as decimal(24,9)) as markup_cost, -- pretax_cost x markup = markup_cost
+    cast(NULL as double) as pod_cost,
+    cast(NULL as double) as project_markup_cost,
+    sum(ocp.pod_usage_cpu_core_hours) as pod_usage_cpu_core_hours,
+    sum(ocp.pod_request_cpu_core_hours) as pod_request_cpu_core_hours,
+    sum(ocp.pod_limit_cpu_core_hours) as pod_limit_cpu_core_hours,
+    sum(ocp.pod_usage_memory_gigabyte_hours) as pod_usage_memory_gigabyte_hours,
+    sum(ocp.pod_request_memory_gigabyte_hours) as pod_request_memory_gigabyte_hours,
+    max(ocp.cluster_capacity_cpu_core_hours) as cluster_capacity_cpu_core_hours,
+    max(ocp.cluster_capacity_memory_gigabyte_hours) as cluster_capacity_memory_gigabyte_hours,
+    max(ocp.pod_labels) as pod_labels,
+    NULL as volume_labels,
+    max(azure.tags) as tags,
+    row_number() OVER (partition by azure.uuid) as project_rank,
```

Comment:
```suggestion
    row_number() OVER (partition by azure.uuid, ocp.data_source) as project_rank,
```

Per https://github.com/project-koku/koku/pull/3402/files